### ISSUE TITLE
onSubmit should close suggestions list

### ISSUE THREAD

**Issue by oliverbienert** [moroshko/react-autosuggest#172]
I want to readress the `closeSuggestions` programmatically issue. I'd like to use the AutoSuggest control like a search input. I may select a value from the list of suggestions but I don't have to. When I press `enter`, with the cursor in the input, the list should close.
I already have wrapped the component in a form:
```
onSubmit(evt) {
       const { callbackFromParent } = this.props;
       callbackFromParent(this.state.value);
}

<form onSubmit={this.onSubmit}>
                <Autosuggest ...
```
This works so far, except that if the list is open and has some values, it does not close, when I just press `enter` in the input (aka submit the form) and do not select a value from the list.
I'd like to have the list being closed when submit is called. Can I somehow fire the `closeSuggestions` action from outside? Or is there another way to programmatically close the list?

Regards Oliver

**Comment by JamieDixon** [comment]
I raised a PR for this exact issue today: https://github.com/moroshko/react-autosuggest/pull/171

**Comment by oliverbienert** [comment]
Yeah, thank you! Would be cool if we can have that.
Cheers Oliver

**Comment by moroshko** [comment]
@oliverbienert I just merged #171 to master, and hope to release a new version soon. Closing as duplicate of #158.
